tests/introspection/iOS/iOSApiSelectorTest.cs

```diff
@@ -158,6 +158,14 @@ protected override bool CheckResponse (bool value, Type actualType, MethodBase m
 			var declaredType = method.DeclaringType;
 
 			switch (declaredType.Name) {
+			case "AVUrlAsset":
+				switch (name) {
+				// fails because it is in-lined via protocol AVContentKeyRecipient
+				case "contentKeySession:didProvideContentKey:":
+					return true;
+				break;
```

Comment:
no need for `break` after a `return`
a `break` would need an extra indentation too